fal_scripts/notify.py

```diff
@@ -0,0 +1,17 @@
+import os
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+
+CHANNEL_ID = os.getenv("SLACK_BOT_CHANNEL")
+SLACK_TOKEN = os.getenv("SLACK_BOT_TOKEN")
+
+client = WebClient(token=SLACK_TOKEN)
+message_text = (
+    f"Model: {context.current_model.name}. Status: {context.current_model.status}."
+)
+
+try:
+    response = client.chat_postMessage(channel=CHANNEL_ID, text=message_text)
+except SlackApiError as e:
+    # You will get a SlackApiError if "ok" is False
+    assert e.response["error"]
```

Comment:
I am not super sure what is happening when this fails. Is it silently existing (if so, maybe we should add something like print("warning: {ENV} is not set, so can't send a message")